Midnight Commander fails to build - Tiger PPC

I did a fresh reinstall of Mac OS X Tiger 10.4.11 and Xcode 2.5. I then installed tigerbrew and per `brew doctor` advise, installed curl and git: `brew install curl git`. Please note, I didn't install Apple GCC 4.2, since I ran into issues with it earlier while trying to install cmake.

I then tried to install Midnight Commander: `brew install mc`, but it fails:

```
/usr/bin/gcc-4.0 -DHAVE_CONFIG_H -I. -I..  -DG_LOG_DOMAIN=\"GLib-GIO\" -I.. -I../glib -I../glib -I.. -I../gmodule -DG_DISABLE_CAST_CHECKS -DGIO_COMPILATION -DGIO_MODULE_DIR=\"/usr/local/lib/gio/modules\"       -I/usr/local/opt/zlib/include -I/usr/local/opt/gettext/include -F/usr/local/Frameworks -D_REENTRANT -Wall -Wstrict-prototypes -Os -w -pipe -mcpu=970 -faltivec -mmacosx-version-min=10.4 -c -o glib-compile-resources.o glib-compile-resources.c
/bin/sh ../libtool  --tag=CC   --mode=link /usr/bin/gcc-4.0 -Wall -Wstrict-prototypes -Os -w -pipe -mcpu=970 -faltivec -mmacosx-version-min=10.4  -L/usr/local/opt/zlib/lib -L/usr/local/opt/gettext/lib -L/usr/local/lib -F/usr/local/Frameworks -Wl,-headerpad_max_install_names -Wl,-framework,Carbon -Wl,-framework,Foundation -o glib-compile-resources gvdb-builder.o glib-compile-resources.o libgio-2.0.la ../gobject/libgobject-2.0.la ../gmodule/libgmodule-2.0.la ../glib/libglib-2.0.la  -lintl  
libtool: link: /usr/bin/gcc-4.0 -Wall -Wstrict-prototypes -Os -w -pipe -mcpu=970 -faltivec -mmacosx-version-min=10.4 -F/usr/local/Frameworks -Wl,-headerpad_max_install_names -Wl,-framework -Wl,Carbon -Wl,-framework -Wl,Foundation -o .libs/glib-compile-resources gvdb-builder.o glib-compile-resources.o  -L/usr/local/opt/zlib/lib -L/usr/local/opt/gettext/lib -L/usr/local/lib ./.libs/libgio-2.0.dylib -L/usr/local/Cellar/pcre/8.39/lib -L/usr/local/Cellar/libffi/3.0.13/lib -L/usr/local/Cellar/zlib/1.2.11/lib /private/tmp/glib20170429-4323-7rqhd4/glib-2.50.1/gobject/.libs/libgobject-2.0.dylib /private/tmp/glib20170429-4323-7rqhd4/glib-2.50.1/gmodule/.libs/libgmodule-2.0.dylib -lz -lresolv ../gobject/.libs/libgobject-2.0.dylib -lffi ../gmodule/.libs/libgmodule-2.0.dylib /private/tmp/glib20170429-4323-7rqhd4/glib-2.50.1/glib/.libs/libglib-2.0.dylib ../glib/.libs/libglib-2.0.dylib /usr/lib/libiconv.dylib -lpcre -lintl
/usr/libexec/gcc/powerpc-apple-darwin8/4.0.1/ld: Undefined symbols:
_deflateSetHeader
_inflateGetHeader
collect2: ld returned 1 exit status
make[4]: *** [glib-compile-resources] Error 1
make[3]: *** [all-recursive] Error 1
make[2]: *** [all] Error 2
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
Error: No such file or directory - /usr/sbin/pkgutil
```

I ran `which ld`, and there is an ld already available. I assume it's from Xcode 2.5. Please help.

After working around <https://github.com/mistydemeo/tigerbrew/issues/506>, I still can't build Midnight Commander, but now for a different reason:

```
$ brew install mc
==> Downloading http://ftp.midnight-commander.org/mc-4.8.14.tar.xz
Already downloaded: /Library/Caches/Homebrew/midnight-commander-4.8.14.tar.xz
==> ./configure --disable-silent-rules --prefix=/usr/local/Cellar/midnight-comma
Last 15 lines from /Users/oboroc/Library/Logs/Homebrew/midnight-commander/01.configure:
checking for GNU gettext in libc... no
checking for GNU gettext in libintl... yes
checking whether to use NLS... yes
checking where the gettext function comes from... external libintl
checking how to link with libintl... -lintl -Wl,-framework -Wl,CoreFoundation
checking for AIX defines... no
configure: using internal editor
configure: using diff viewer
checking for subshell support... yes
configure: Enabling VFS code
checking for zip... /usr/bin/zip
checking for unzip... /usr/bin/unzip
checking for zipinfo code in unzip... yes
checking for LIBSSH... no
configure: error: libssh2 >= 1.2.5 library not found
```

--- a/Library/Formula/midnight-commander.rb
+++ b/Library/Formula/midnight-commander.rb
@@ -1,19 +1,11 @@
 class MidnightCommander < Formula
   desc "Terminal-based visual file manager"
   homepage "https://www.midnight-commander.org/"
-  url "http://ftp.midnight-commander.org/mc-4.8.14.tar.xz"
-  mirror "https://mirrors.ocf.berkeley.edu/debian/pool/main/m/mc/mc_4.8.14.orig.tar.xz"
-  sha256 "6b1f74efbdf394bfab084f4d2ea2e72173de5f12cd42be2533ca2c3b72edb8e6"
+  url "http://ftp.midnight-commander.org/mc-4.8.29.tar.xz"
+  mirror "https://mirrors.ocf.berkeley.edu/debian/pool/main/m/mc/mc_4.8.29.orig.tar.xz"
+  sha256 "01d8a3b94f58180cca5bf17257b5078d1fd6fd27a9b5c0e970ec767549540ad4"
 
   head "https://github.com/MidnightCommander/mc.git"
-
-  bottle do
-    revision 1
-    sha256 "e9f5109f2fd2927096b9b6cab05c5509695b313b072126c6f352d7ba3891f748" => :el_capitan
-    sha256 "9e25d3d107929070400bcf8f928e5adc3f77c6f4942544cbf7b26822ecc425d6" => :yosemite
-    sha256 "faa3115190a66947d3141619d285c55fac451cd154be9a6ea053b37239ecd085" => :mavericks
-    sha256 "e75f3aed5a28a381a1bce9461108d21e76abb094d98ad8cfe580720ad390d7e8" => :mountain_lion
-  end
 
   depends_on "pkg-config" => :build
   depends_on "glib"
@@ -22,17 +14,13 @@ class MidnightCommander < Formula
   depends_on "libssh2"
 
   def install
-    system "./configure", "--disable-debug",
-                          "--disable-dependency-tracking",
+    system "./configure", "--disable-dependency-tracking",
                           "--disable-silent-rules",
                           "--prefix=#{prefix}",
                           "--without-x",
                           "--with-screen=slang",
                           "--enable-vfs-sftp"
     system "make", "install"
-
-    # https://www.midnight-commander.org/ticket/3509
-    inreplace libexec/"mc/ext.d/text.sh", "man -P cat -l ", "man -P cat "
   end
 
   test do

